### PR TITLE
Added instance of mean for numpy frontend

### PR DESCRIPTION
**Pull request by Hassan-Ik**

### CODE REVIEW

**Comment by AnnaTz** [comment]
Hi there, if this is part of a ToDo list issue, could you please make a comment "Close #issue_number" to link it?

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 1]
```diff
@@ -294,6 +294,48 @@ def test_numpy_instance_argsort(
...
 
+@handle_cmd_line_args
+@given(
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+    keepdims=st.booleans(),
+    where=np_frontend_helpers.where(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.ndarray.mean"
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:305]: The test fails with `AttributeError: module 'ivy.functional.frontends.numpy.ndarray' has no attribute 'mean'`. I think the `fn_name` should be `ivy.functional.frontends.numpy.ndarray.ndarray.mean`. 
You can check out errors by clicking `Details` next to 
`test-frontend-numpy / run-nightly-tests (pull_request)`. Then you should expand `Run Frontend Tests` and search for your test's name.

**Review by AnnaTz — CHANGES_REQUESTED** [review]
Please be careful to not commit changes in the `.idea` folder. You should rollback the 3 `.xml` files.

**Comment by AnnaTz** [comment]
Hi there, the PR was almost ready! Why did you close it?

**Comment by Hassan-Ik** [comment]
Yes, Sorry about that actually I reverted commits and pushed those changes on remote git repo too due to which my PR was closed.

**Comment by Hassan-Ik** [comment]
Yes, Sorry about that actually I reverted commits and pushed those changes on remote git repo too due to which my PR was closed.

**Pull request closed without merging**